templates/overrides/views/views-view-unformatted.html.twig

```diff
@@ -0,0 +1,22 @@
+{#
+/**
+ * @file
+ * Template for a unformatted rows.
```

Comment:
The english here is not correct, but I would change it completely to "Theme override to display a view of unformatted rows."